public/static/locales/en/cognito-errors.json

```diff
@@ -16,6 +15,12 @@
       "title": "To reset your password, you must first verify your email."
     }
   },
+  "TooManyRequestsException": {
```

Comment:
I re-structured this one to handle the support link -- fr needs to be updated.

All errors in this cognito file will handle links --- they need to be setup to do so.

See this commit where the json is modified and we update the `useAuthErrors.ts` hook to look for the text.


https://github.com/cds-snc/platform-forms-client/pull/2846/commits/bae30981ffe3605dbf41d09196f6152748782dc6